scripts/release/update-package-versions.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+// Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in `packages/*` and `release/*` directories to the latest version.
+// Changes in `packages/*` will be committed as well.
+//
+// See https://github.com/cksource/ckeditor5-internal/issues/1123.
+//
+// This task must be called before: `npm run release:publish`.
+//
+// Use:
+// npm run release:update-package-versions -- --dry-run
+//
+// Available arguments:
+// --dry-run:   Prevents the script from committing, and instead displays detailed list of changes from each file that was changed.
+
+const CKEDITOR5_PATH = path.resolve( __dirname, '..', '..' );
+const CKEDITOR5_INTERNAL_PATH = path.resolve( __dirname, '..', '..', 'external', 'ckeditor5-internal' );
+const COLLABORATION_FEATURES_PATH = path.resolve( __dirname, '..', '..', 'external', 'collaboration-features' );
```

Comment:
Minor idea to consider: paths to `ckeditor5-internal` and `collaboration-features` could be now based on the `CKEDITOR5_PATH` constant, so when the location of this script changes in the future, only one path to the CKEditor 5 will have to be adapted.